lib/smart_app_launch/backend_auth_request_builder.rb

```diff
@@ -0,0 +1,72 @@
+require 'json/jwt'
+require_relative 'client_assertion_builder'
+
+module SMARTAppLaunch
+  class BackendServicesAuthorizationRequestBuilder
```

Comment:
All of these file names should be the lower snake case version of the class name.

Reply:
I updated all the file names (though some of them are pretty long now, hopefully that's okay).